packages/attestation-service/src/requestHandlers/get_attestation.ts

```diff
@@ -2,13 +2,13 @@ import { PhoneNumberUtils } from '@celo/utils'
 import { GetAttestationRequest } from '@celo/utils/lib/io'
 import Logger from 'bunyan'
 import express from 'express'
-import { findAttestationByKey, makeSequelizeLogger, SequelizeLogger } from '../db'
-import { AttestationKey } from '../models/attestation'
-import { respondWithAttestation, respondWithError } from '../request'
+import { Transaction } from 'sequelize'
+import { findAttestationByKey, makeSequelizeLogger, sequelize, SequelizeLogger } from '../db'
+import { AttestationKey, AttestationModel } from '../models/attestation'
+import { ErrorWithResponse, respondWithAttestation, respondWithError } from '../request'
 import { obfuscateNumber } from '../sms/base'
 
-export const VERSION = process.env.npm_package_version as string
-export const SIGNATURE_PREFIX = 'attestation-service-status-signature:'
+const maxSecurityCodeAttempts = 5
```

Comment:
Nit: Convention calls for the constant names to be capitalized. Also there seems to be enough constants now that it would maybe make sense to have a centralized `constants.ts` file?